Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -689,6 +587,197 @@ script:
+      description: The standard output of the command.
+      type: string
+    - contextPath: CrowdStrike.Command.Stderr
+      description: The standard error of the command.
+      type: string
+    - contextPath: CrowdStrike.Command.BaseCommand
+      description: The base command.
+      type: string
+    - contextPath: CrowdStrike.Command.Complete
+      description: True if the command has done.
+      type: boolean
+    - contextPath: CrowdStrike.Command.SequenceID
+      description: The sequence ID in the current request.
+      type: number
+    - contextPath: CrowdStrike.Command.NextSequenceID
+      description: The sequence ID for the next request  in the chunk request.
+      type: number
+    description: Get status of an executed command on a host
+  - name: cs-falcon-get-extracted-file
+    arguments:
+    - name: host_id
+      required: true
+      description: The host agent ID.
+    - name: sha256
+      required: true
+      description: The SHA-256 hash of the file.
+    - name: filename
+      description: Filename to use for the archive name and the file within the archive.
+    description: Get RTR extracted file contents for specified session and sha256.
+  - name: cs-falcon-list-host-files
+    arguments:
+    - name: host_id
+      required: true
+      description: The ID of the host agent of which it lists files in the session
+    outputs:
+    - contextPath: CrowdStrike.Command.HostID
+      description: The ID of the host for which the command was running.
+      type: string
+    - contextPath: CrowdStrike.Command.TaskID
+      description: The ID of the command request which has been accepted.
+      type: string
+    - contextPath: CrowdStrike.Command.SessionID
+      description: The ID of the session of the host.
+      type: string
+    - contextPath: CrowdStrike.File.ID
+      description: The ID of the file.
+      type: string
+    - contextPath: CrowdStrike.File.CreatedAt
+      description: The creation date of the file.
+      type: date
+    - contextPath: CrowdStrike.File.DeletedAt
+      description: The deletion date of the file.
+      type: date
+    - contextPath: CrowdStrike.File.UpdatedAt
+      description: The last updated date of the file.
+      type: date
+    - contextPath: CrowdStrike.File.Name
+      description: The full file name.
+      type: string
+    - contextPath: CrowdStrike.File.SHA256
+      description: The SHA-256 hash of the file.
+      type: string
+    - contextPath: CrowdStrike.File.Size
+      description: The size of the file in bytes.
+      type: number
+    - contextPath: File.Name
+      description: The full file name.
+      type: string
+    - contextPath: File.Size
+      description: The size of the file in bytes.
+      type: number
+    - contextPath: File.SHA256
+      description: The SHA-256 hash of the file.
+      type: string
+    description: Get a list of files for the specified RTR session on a host.
+  - name: cs-falcon-refresh-session
```

Comment:
why is this command needed?

Reply:
Implemented it with customer request. I didn't ask them why they need it.